src/migrations/20190626072921-add-delayed-author-reward-to-block.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+        return queryInterface.addColumn("Blocks", "delayedAuthorReward", {
+            type: Sequelize.NUMERIC({ precision: 20, scale: 0 }),
+            defaultValue: "0"
+        });
+    },
+
+    down: (queryInterface, Sequelize) => {
+        return queryInterface.removeColumn("Blocks", "delayedAuthorReward");
```

Comment:
Is it the same thing with the intermediate reward?

Reply:
Yes, the same thing. Is it better to rename the column name to "intermedateReward"?

Reply:
I changed the column name to "intermediateRewards"